src/stories/layouts/HorizontalLayout.stories.tsx

```diff
@@ -5,28 +5,32 @@ import {
   numericMagnitudeSchema,
   numericHorizontalUISchema,
 } from "../../testSchemas/numericSchema/numericSchema"
+import {
+  numericSliderBasisPointsSchema,
+  numericSliderHorizontalUISchema,
+} from "../../testSchemas/numericSchema/numericSliderSchema"
 
 const meta: Meta<typeof StorybookAntDJsonForm> = {
   title: "Layout/Horizontal",
   component: StorybookAntDJsonForm,
   tags: ["autodocs"],
-  args: {
-    uiSchema: numericHorizontalUISchema,
-  },
-  argTypes: {
-    uiSchema: {
-      control: "object",
-    },
-  },
 }
 
 export default meta
 type Story = StoryObj<typeof StorybookAntDJsonForm>
 
-export const NumericControl: Story = {
+export const NumericInput: Story = {
   tags: ["autodocs"],
   args: {
     jsonSchema: numericMagnitudeSchema,
     uiSchema: numericHorizontalUISchema,
```

Comment:
if the story is illustrating horizontal (or vertical) layouts, doesn't the ui schema need more than one field to illustrate the difference?

Reply:
There is an allowable width component as demonstrated in Storybook:
<img width="988" alt="image" src="https://github.com/great-expectations/jsonforms-antd-renderers/assets/19361723/60df019e-8596-4317-88e2-3cd0c604b0ff">
vs
<img width="988" alt="image" src="https://github.com/great-expectations/jsonforms-antd-renderers/assets/19361723/e5dbdb90-9e78-4542-b3db-d3d83ad44335">